app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,21 @@
             </a>
           </div>
           <div class="col d-flex justify-content-end">
+            <% if user_signed_in? %>
+              <nav class="userNav">
+                <div class="d-flex">
+                  <div class="left user-avatar">
+                    <a href="#">
+                      <%= image_tag "icons/user-avatar.svg" %>
+                    </a>
+                  </div>
+                  <div class="right user-name">
+                    <p><a class="uname-link primary" href="#"><strong><%= "#{current_user.given_name} #{current_user.last_name}"  %></strong></a></p>
+                    <p><small><%= link_to raw("Cerrar sesi&oacute;n"), destroy_user_session_path, class: "signout-link" %></small></p>
```

Comment:
```suggestion
                    <p><small><%= link_to raw("Cerrar sesi&oacute;n"), destroy_user_session_path, method: :delete, class: "signout-link" %></small></p>
```
Creo que eso debe de arreglarlo

https://github.com/plataformatec/devise/wiki/How-To:-Add-sign_in,-sign_out,-and-sign_up-links-to-your-layout-template

Reply:
Beautiful. Una disculpilla hahaha